src/main/java/cc/polyfrost/oneconfig/config/elements/BasicOption.java

```diff
@@ -91,7 +92,7 @@ public Object get() throws IllegalAccessException {
      * @param x  x position
      * @param y  y position
      */
-    public abstract void draw(long vg, int x, int y);
+    public abstract void draw(long vg, int x, int y, InputHandler inputHandler);
```

Comment:
I don't exactly like this but I can't see any way around it that doesn't defeat it's purpose

Reply:
Well yeah if it’s not static then it could cause conflicts because of the scale stuff so I think this is the best opion